src/NzbDrone.Core/MediaFiles/MediaInfo/MediaInfoFormatter.cs

```diff
@@ -107,8 +128,13 @@ public static string FormatAudioCodec(MediaInfoModel mediaInfo, string sceneName
                 return "PCM";
             }
 
-            if (audioFormat.EqualsIgnoreCase("TrueHD"))
+            if (audioFormat.EqualsIgnoreCase("MLP FBA"))
```

Comment:
I dunno, it's not there in radarr, and certainly the (admittedly small number of) TrueHD episode files i looked at all had `MLP FBA`. I guess it doesn't hurt to leave it in, just in case.

Reply:
Yes please, it's possible it only happened with older mediainfo versions, but I rather not take chances given that it doesn't hurt to leave it in.

Reply:
Dolby TrueHD uses Meridian Lossless Packing (MLP) as its mathematical basis for compressing audio samples.
https://www.dolby.com/us/en/technologies/mlp-lossless.html

now from the mediainfo peeps,
> We do the difference between technical name and the marketing name, e.g. "E-AC-3 JOC" for Dolby Atmos in E-AC-3, "MLP FBA" for TrueHD and "MLP FBA 16ch" for Atmos in TrueHD.

so personally I'd say stick to the marketing name since that is what consumers will know :)

Reply:
I've added the TrueHD check back in